--- python-zeroconf-0.19.1/test_zeroconf.py.orig	2017-06-13 08:34:17.000000000 +0200
+++ python-zeroconf-0.19.1/test_zeroconf.py	2023-03-26 17:55:15.249452986 +0200
@@ -272,7 +272,10 @@ class Names(unittest.TestCase):
         assert longest_packet[0] >= r._MAX_MSG_ABSOLUTE - 100
 
         # mock zeroconf's logger warning() and debug()
-        from mock import patch
+        try:
+            from mock import patch
+        except ImportError:
+            from unittest.mock import patch
         patch_warn = patch('zeroconf.log.warning')
         patch_debug = patch('zeroconf.log.debug')
         mocked_log_warn = patch_warn.start()
